apps/sms/views/conversation/js/conversation.js

```diff
@@ -2172,7 +2172,7 @@ var ConversationView = {
 
   sendMessage: function conv_sendMessage(opts) {
     var content = Compose.getContent(),
-        subject = Compose.getSubject(),
+        subject = Compose.isSubjectVisible ? Compose.getSubject() : null,
```

Comment:
Oh, it was my first implementation before the draft part conflict actually ;) @azasypkin  How do you  fell about getSubject checks the visibility inside compose?